index2.js

```diff
@@ -0,0 +1,111 @@
+//
```

Comment:
Hanging onto this may be helpful for dev purposes but you should delete it before committing.